crates/smartmodule-development-kit/tests/cmd.rs

```diff
@@ -0,0 +1,32 @@
+use std::process::Command;
```

Comment:
`smartmodule-development-kit` is a CLI application and therefore it seems appropriate to verify what the user will actually see.